scripts/missions/asa/02_Burgeoning_Dread.lua

```diff
@@ -0,0 +1,106 @@
+-----------------------------------
+-- A Shantotto Ascension
+-- A Shantotto Ascension M2
+-----------------------------------
+-- !addmission 11 1
+-----------------------------------
+require('scripts/globals/missions')
+require('scripts/settings/main')
+require('scripts/globals/interaction/mission')
+require('scripts/globals/zone')
+-----------------------------------
+
+local mission = Mission:new(xi.mission.log_id.ASA, xi.mission.id.asa.BURGEONING_DREAD)
+
+mission.reward =
+{
+    nextMission = { xi.mission.log_id.ASA, xi.mission.id.asa.THAT_WHICH_CURDLES_BLOOD },
+}
+
+mission.sections =
+{
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return currentMission == mission.missionId
+        end,
+
+        [xi.zone.EAST_SARUTABARUTA] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    if
+                        prevZone == xi.zone.WINDURST_WOODS and
+                        player:hasStatusEffect(xi.effect.MOUNTED) == false
+                    then
+                        return mission:event(71)
+                    end
+                end,
+            },
+
+            onEventUpdate =
+            {
+                [71] = function(player, csid, option, npc)
+                    local kit = 2779 + math.random(0,3)
+                    player:updateEvent(kit)
+                    player:setCharVar("ASA_kit", kit)
```

Comment:
`player:setCharVar("ASA_kit", kit)`

So this has been bugging me a bit since I saw it, and I think there's a reasonable way to use a mission var for this instead, and that is to set the mission var for the _next_ mission, since that is where it is used after the eventUpdate here.  To do this, use the xi.mission.setVar function like below:

`xi.mission.setVar(player, xi.mission.log_id.ASA, xi.mission.id.asa.THAT_WHICH_CURDLES_BLOOD, 'Option', kit)`

There's a few standard variables we use in missions and quests for the most part (Status, Prog, Option, and Timer being a few of them)